### DISCUSSION TITLE
Unable to import scenarios

### DISCUSSION THREAD

**Discussion by Mastomaki** [Spine-project/Spine-Toolbox#2938] [Support]
I have to run my scenario importer twice before it works. I guess the reason is that `scenario_alternative` is imported before alternatives. How to change the order of importing?

![image](https://github.com/user-attachments/assets/37208f22-5dfb-4685-8d61-4e2edb37f250)

**Comment by soininen** [comment]
You can drag the tables around in the list with mouse to change their order.

**Comment by Mastomaki** [comment]
Ok, now I got it working.
